apps/devtools/package.json

```diff
@@ -46,11 +46,14 @@
     "graphiql": "^1.3.2",
     "graphql": "^15.5.0",
     "graphql-tag": "^2.11.0",
+    "jotai": "^1.5.3",
     "prism-react-renderer": "^1.2.1",
     "prop-types": "^15.7.2",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-scribble": "1.1.4",
```

Comment:
Scribble? lol

Reply:
this is a library I created to integrate canvas into React. TBH it's not that great, and I had some bugs with it in this project. That being said it manages the render loop pretty well and exposes everything pretty nicely in the draw functions. Actually probably going to remove it in the future because I am actually avoiding some of the features I initially introduced it for.